### PR TITLE
[FIX] Setting correct path to AdapterPBS config file

### PR DESCRIPTION
**Pull request by avincigu**

### CODE REVIEW

File: resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java [hunk 1]
```diff
@@ -76,7 +78,10 @@ public long handleInputFromExternalComponent(String[] aWiParms) throws ProviderE
             HashMap<String, String> args = new HashMap<String, String>();
 
             String configName = AdapterWlmPBS.class.getSimpleName() + ".json";
-            PropertyMap configJson = jsonParser.readConfig(configName).getAsMap();
+            xdg = new XdgConfigFile("ucs");
+            InputStream result = xdg.Open(configName);
+            
+            PropertyMap configJson = jsonParser.readConfig(result).getAsMap();
```
Comment (paul-amonson) [inline resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java:84]: Does readConfig check for a null "result" stream? It is possible to get back null from the xdg.Open(). If not you need to handle this fatal condition.

File: resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java [hunk 2]
```diff
@@ -99,6 +104,7 @@ public long handleInputFromExternalComponent(String[] aWiParms) throws ProviderE
             String instancedata = "AdapterName=" + adapter.getName();
             raseventlog.logRasEventSyncNoEffectedJob(eventtype, instancedata, null, System.currentTimeMillis() * 1000L, adapter.getType(), workQueue.workItemId());
             log_.error("Unable to connect to network sink");
+            log_.exception(e);
```
Comment (paul-amonson) [inline resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java:107]: Why 2 calls to log_?  Instead use:  log_.exception(e, "Unable to connect to network sink");

File: resource_managers/src/test/groovy/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.groovy [hunk 3]
```diff
@@ -169,6 +170,10 @@ class AdapterWlmPBSSpec extends Specification {
             put("topics","pbs_runjobs")}}
         jsonparser_.readConfig(_) >> configJson
         underTest_.jsonParser = jsonparser_
+        def stream = Mock(InputStream)
+        def xdg_ = Mock(XdgConfigFile)
+        xdg_.Open(_) >> stream
+        underTest_.xdg = xdg_
```
Comment (paul-amonson) [inline resource_managers/src/test/groovy/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.groovy:176]: Nice!

**Review by paul-amonson — CHANGES_REQUESTED** [review]
A couple of changes.

File: resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java (hunk 2 above)
Comment (avincigu) [reply resource_managers/src/main/java/com/intel/dai/resource_managers/pbs/AdapterWlmPBS.java]: Fixed

**Review by paul-amonson — APPROVED** [review]

**Review by sindhu-106 — APPROVED** [review]

**Pull request merged**